main.html

```diff
@@ -1,77 +1,252 @@
-<h1>A Simple Oriole</h1>
```

Comment:
All of this red text is the starter "template" text.  You can just ignore it.  Only look at the green (added) text.